test/suite/test_hs01.py

```diff
@@ -136,17 +143,21 @@ def test_hs(self):
         session2.rollback_transaction()
         session2.close()
 
-        # Scenario: 3
-        # Check to see if the history store is working with the old timestamp.
-        bigvalue4 = b"ddddd" * 100
-        self.conn.set_timestamp('stable_timestamp=' + timestamp_str(1))
-        self.large_updates(self.session, uri, bigvalue4, ds, nrows, timestamp=True)
-        # Check to see data can be see only till the stable_timestamp
-        self.durable_check(bigvalue3, uri, ds, nrows)
+        # Don't run durable check for columnar store. Since durable history rollback is 
+        # not implemented for columnar store. Remove this if statement once WT-5545
```

Comment:
What do you mean by durable history rollback? We never rollback things in history store.

Reply:
Sorry, I mean rollback to stable for the history store during recovery isn't implemented for the column store. @kommiharibabu

Reply:
Yes, Rollback to stable support for column store is not implemented. Due to this reason, any tests that operate with timestamps and later verifying the data after recovery on column stores can fail.

@Sean04 You need to update the comment to mention the `timestamps`. The durable check is already used in the above scenarios, but it fails only when it is used with timestamps. Also, change `durable history rollback` to `rollback to stable`.